apps/public-docsite/src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -70,15 +70,15 @@ export class NotFoundPage extends React.Component<INotFoundPageProps, {}> {
   ];
 
   /** Gets the top level page from the current URL and returns a link to it. */
-  private _getAreaLink = (): JSX.Element => {
+  private _getAreaLink = (): JSX.Element | undefined => {
```

Comment:
lot of this pattern around the website codebase. function that returns nothing but is annotated as return always a type